app/views/legislation/processes/_process.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="small-12 medium-8 column">
       <div class="legislation-text">
         <h3><%= link_to process.title, process %></h3>
-        <p><%= process.description %></p>
+        <%= simple_format(first_paragraph(process.description)) %>
```

Comment:
I think this should use markdown instead of simple_format, because the description field can use markdown (this wasn't addressed before either)